Dockerfile.ziphq

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.17.1 as builder
```

Comment:
Curious why this file has ziphq file extension?

Reply:
because it already has `Dockerfile`, `Dockerfile.dev`, need a new name.